src/Transition/VCompareGlobalRoutines.f90

```diff
@@ -778,33 +778,63 @@ SUBROUTINE CheckSpecialObjects(DifUnit,ObjectName,CurArgs,OutArgs,FieldNames,Fie
           WRITE(DifUnit,fmta) ''
 
         CASE('FENESTRATIONSURFACE:DETAILED')
-          CALL WriteOutPartialIDFLines(DifUnit,ObjectName,9,OutArgs,FieldNames,FieldUnits)
-          IF (MakeUPPERCase(OutArgs(9)) == 'AUTOCALCULATE') THEN
-            NVert=(CurArgs-9)/3
-          ELSEIF (OutArgs(9) == '') THEN
-            NVert=(CurArgs-9)/3
-          ELSE
-            READ(OutArgs(9),*) NVert
-          ENDIF
-          VArg=10
-          DO Arg=1,NVert
-            IF (Arg /= NVert) THEN
-              LString=',  !- '
+          IF (VersionNum < 9.0) THEN
```

Comment:
These diffs are best viewed with white space ignored. Bascially, if VersionNum< 9.0 use the same block as BuildingSurface:Detailed above, otherwise, else use the new block for the shorter fenestration object.